### ISSUE TITLE
Feedback on material published.  Placegholder.

### ISSUE THREAD

**Issue by weynandkuijpers** [threefoldfoundation/info_threefold#190]
### Issues found
- The amount of TFT in circulation in the market is however limited to a maximum of 4 billion TFT.
Thats a good place to start, but several other places in the documents, primarily in the screenshots, I noticed i says the maximum is 2 billion. Perhaps some sections of your documents are out of date. However it creates ambiguity about the actual supply hard cap if it is not stated consistently. 
- Farming difficulty follows the logic of diminishing returns, meaning that over time, less and less TFT are earned.
Okay, this sounds similair to a pow based system. There is no hard equation here to reference though, so again ambiguous.
- To maximize the stability of farming returns, the amount of TFTs earned for farming is normalized against the US Dollar.
Does this mean the amount of coins created is not at a fixed predictable rate but is instead adjusted based on the exchange rate? If so, does this change the speed of coin production / inflation?
- Providing hardware reliably is rewarded with a certification that leads to increased earnings in TFT for certified farmers.
This seems to be another fact that changes rewards. It would be helpful to have all facts about coin creation summarized into a single section. Currently this information is dispersed across several documents.
- X % is kept for foundation over x months.
This is pointing to another fact of distribution that is not clarrified and leaves ambiguity.
- Burning 50% of cultivated tokens?
Same, not sure what this is referring to either but it sounds like something that effects the coin supply.
- Today 800m Tokens have been farmed (mined) from which only 73 million are tradable.
This fact is giving more information about the existing supply which is helpful. If 800m tokens exist, and if 4 billion is the hard cap, that would mean that 20% of coins have already been distributed. More clarity is needed on how these were distributed.
- Locked tokens
Mentioned are made about locked tokens. More clarification is needed.
- Stellar distributions event, and other distribution events?
This points to another method of distribution. How are these distribution events controlled? Are coin distributions made through a centrally controlled method? 
- Other tokens?  "Each project has its own branded token"
It seems there a sub-tokens of a sort. I see no explanation for how these are created or what there relation is to the primary token, such as if they have an effect on the supply.

### Summary feedback
In summary, it appears some information on the coin distribution is available, but it is lacking clear organization in some areas, there are some areas where explanation is missing, and for myself its not currently possible to get an objective view of the technical or economic functioning.

**Comment by despiegk** [comment]
duplicate #189
